### PR TITLE
add doom

### PR DESCRIPTION
**Pull request by Ridleh**
fun card, much counterplay

### CODE REVIEW

File: devTools/cardCreator/cards_to_import/Doom.spell.card.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        "makeStun": false,
+        "makeDisarm": false,
+        "noDisarm": false,
+        "noPoison": false,
+        "noStun": false,
+        "noBadEffect": false,
+        "noAttackFromWeakerOnes": false,
+        "disableHolyBuff": false,
+        "addSpell": false,
+        "killsTarget": false,
```
Comment (Polynucleotide) [inline devTools/cardCreator/cards_to_import/Doom.spell.card.json]:
```suggestion
        "hpChange": 0,
        "mpChange": 0,
        "poison": false,
        "makeStun": false,
        "makeDisarm": false,
        "noDisarm": false,
        "noPoison": false,
        "noStun": false,
        "noBadEffect": false,
        "noAttackFromWeakerOnes": false,
        "disableHolyBuff": false,
        "addSpell": false,
        "killsTarget": true,
```

Comment (fluffy-hamster) [reply devTools/cardCreator/cards_to_import/Doom.spell.card.json]: If kills target works on heros, this is probably a better change.

Comment (Ridleh) [reply devTools/cardCreator/cards_to_import/Doom.spell.card.json]: done

**Comment by Ridleh** [comment]
should be good now

**Pull request merged**